### PR TITLE
Addign K8s realease notes for 24.01

### PR DESCRIPTION
**Pull request by saynb**

### CODE REVIEW

File: RELEASENOTES.md [hunk 1]
```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
   * Supports maximum of 2 vports and each port supports maximum of 4 rx and tx
    queues
 
+### K8s Recipe
```
Comment (5abeel) [inline RELEASENOTES.md:80]: To keep it consistent with previous releases, please update to call this

`Kubernetes Infrastructure Offload Recipe`

File: RELEASENOTES.md [hunk 2]
```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
...
+#### Features and enhancements
+
+* Service Load Balancing: Support for K8s Service of type ClusterIP.
+   Kubeproxy implementation is now offloaded to hardware.
+   Services can be created and dynamically distributed to endpoints.
+   For TCP, only the SYN packet goes through the load-balancing logic.
+   Entry is added to hardware CT table for treatment of subsequent packets.
+   Support for dynamic scale-up of endpoints.
+* Support for Go version 1.21.4
+* Support for log level configuration from config files for Infraagent
```
Comment (5abeel) [inline RELEASENOTES.md:91]: Is "infra agent" two words? or should the word be PascalCase?

File: RELEASENOTES.md [hunk 3]
```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
...
+* SRIOV support for ES2100
```
Comment (5abeel) [inline RELEASENOTES.md:92]:
```suggestion
* SRIOV support for Intel IPU E2100
```

**Review by 5abeel — COMMENTED** [review]
A few minor comments to be addressed

**Review by 5abeel — APPROVED** [review]
LGTM

File: RELEASENOTES.md [hunk 4]
```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
...
+### Kubernetes Infrastructure Offload Recipe
+
+#### Features and enhancements
+
+* Service Load Balancing: Support for K8s Service of type ClusterIP.
+   Kubeproxy implementation is now offloaded to hardware.
+   Services can be created and dynamically distributed to endpoints.
+   For TCP, only the SYN packet goes through the load-balancing logic.
+   Entry is added to hardware CT table for treatment of subsequent packets.
+   Support for dynamic scale-up of endpoints.
```
Comment (shweta503) [inline RELEASENOTES.md:89]: Caching applies to both TCP and UDP. I would suggest change lines 86-88 to the following.
"TCP and UDP services can be created and are dynamically distributed to endpoints.
The first packet of each flow goes through load balancing logic, and the result is
cached in hardware auto-add table for treatment of subsequent packets, resulting in 
increased efficiency."

**Review by venkateshpnv — APPROVED** [review]

**Review by dandaly — APPROVED** [review]

**Review by shweta503 — APPROVED** [review]

**Pull request merged**